lib/widget/visualization/chart_js.dart

```diff
@@ -136,6 +137,10 @@ class ChartJsState extends WidgetState<ChartJs> {
 
   @override
   Widget buildWidget(BuildContext context) {
+    if (Platform.isWindows) {
```

Comment:
does it even work outside Web? if it doesn't just do this for everything except kIsWeb

Reply:
Yes, it work on web, android and IOS. I was under the impression it works as well on macOS but it doesn't so I'll update the condition.